### PR TITLE
Added check if media exists in isPaused function

### PR DESCRIPTION
**Pull request by oroshnivskyy**
Added check if media exists in isPaused mediaelement function, it's useful when we trying stop audio when it was destriyed or not set

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" />
+</project>
```
Comment (thijstriemstra) [inline .idea/misc.xml]: this shouldn't be part of the PR.

Comment (oroshnivskyy) [inline .idea/misc.xml]: Sorry, i updated PR.

File: src/mediaelement.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ WaveSurfer.util.extend(WaveSurfer.MediaElement, {
     },
 
     isPaused: function () {
-        return this.media.paused;
+        return this.media && this.media.paused;
```
Comment (oroshnivskyy) [inline src/mediaelement.js]: @katspaugh  Is it ok to return false when media not exists?

Comment (katspaugh) [inline src/mediaelement.js]: @oroshnivskyy I'm not even sure we are taking the right approach hiding all these possible errors in user's application. Why can't the user destroy the instance cleanly in their code? And not call `isPaused` when they know it's destroyed. Or is it called internally?

Comment (oroshnivskyy) [inline src/mediaelement.js]: @katspaugh It was called internally. How do you think if we will create [Null Object](https://sourcemaking.com/design_patterns/null_object) and set it to media when media it is destroued. Player will work like a cassete player when you turn it on without cassete.

Comment (katspaugh) [inline src/mediaelement.js]: If it's called internally, then I guess a check is enough. As for returning `false`, I would expect `isPaused` to return `true` if it's not playing.

**Comment by katspaugh** [comment]
@oroshnivskyy can you update this PR so that `isPaused` should return `true` when there's no `media`?

**Comment by oroshnivskyy** [comment]
@katspaugh yes sorry for delay

**Comment by oroshnivskyy** [comment]
@katspaugh Hello i updated PR

**Comment by katspaugh** [comment]
Thanks! Could you add spaces around the `||` operator? Like this: `!this.media || this.media.paused;`

**Comment by oroshnivskyy** [comment]
@katspaugh Updated PR

**Comment by katspaugh** [comment]
:ok_hand:
